baldrick/github/github_api.py

```diff
@@ -2,22 +2,22 @@
 import base64
 import re
 import requests
-import warnings
 from datetime import datetime
 
 import dateutil.parser
 from flask import current_app
+from loguru import logger
 from ttldict import TTLOrderedDict
 
-from baldrick.config import loads
+from baldrick.config import loads, Config
 from baldrick.github.github_auth import github_request_headers
 
 __all__ = ['GitHubHandler', 'IssueHandler', 'RepoHandler', 'PullRequestHandler']
 
 HOST = "https://api.github.com"
 HOST_NONAPI = "https://github.com"
 
-cfg_cache = TTLOrderedDict(default_ttl=60 * 60)
+FILE_CACHE = TTLOrderedDict(default_ttl=60)
```

Comment:
While you are at it, why not make `default_ttl` configurable?

Reply:
I tried, but it can't use `app.config` as we can't get the app context at import time. The best I could do is an env var?

Reply:
Env var is better than nothing, I guess. :+1:

Reply:
Done. We really need a page in the docs for deployment configuration. We should probably make the Heroku page not Heroku specific, it's mostly true however you deploy the thing. A job for a different PR.